### ISSUE TITLE
Battery status

### ISSUE THREAD

**Issue by Tamas92** [fasmide/deflux#6]
Hi,

First of all it is not an issue it should be a suggestion or upgrade request.
Deflux works fine for me, thanks for your hard work. Before the Conbee stick I used zigbee2mqtt and there the battery status was in the query as well. Is it possible to put it to the query as well?

**Comment by fasmide** [comment]
Hi

This sounds like a great idea - do you happen to know if deconz provides these metrics? - I think i would have spotted them if it did - then again - its been quite a while since I looked at its API's and the stream of data coming from it :)

**Comment by Tamas92** [comment]
Yes, deconz is showing the battery status, so the data is there.
